core/src/components/split-pane/split-pane.tsx

```diff
@@ -112,17 +112,15 @@ export class SplitPane implements ComponentInterface {
       return;
     }
 
-    if ((window as any).matchMedia) {
-      // Listen on media query
-      const callback = (q: MediaQueryList) => {
-        this.visible = q.matches;
-      };
-
-      const mediaList = window.matchMedia(mediaQuery);
-      (mediaList as any).addListener(callback as any);
-      this.rmL = () => (mediaList as any).removeListener(callback as any);
-      this.visible = mediaList.matches;
-    }
+    // Listen on media query
+    const callback = (q: MediaQueryList) => {
+      this.visible = q.matches;
+    };
+
+    const mediaList = window.matchMedia(mediaQuery);
+    (mediaList as any).addListener(callback as any);
```

Comment:
Unrelated to your changes, but `addListener` and `removeListener` are [deprecated APIs](https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList/addListener). Thoughts on tracking a technical debt to resolve those or just updating them?

I believe it would be to the effect of:
```ts
const callback = (q: MediaQueryListEvent) => {
  this.visible = q.matches;
};

const mediaList = window.matchMedia(mediaQuery);
mediaList.addEventListener('change', callback);
this.rmL = () => mediaList.removeEventListener('change', callback);
```

Reply:
Good idea. I don't have time at the moment to test and verify there are no behavior differences (there shouldn't be, but good to check just in case). I am tracking this in a ticket now.